### PR TITLE
Fix #1398: Set minimum image size to 48dp

### PR DESCRIPTION
**Pull request by rt4914**
## Explanation

Fixes #1398

This PR makes sure that the image size in rich-text is atleast 48dpx48dp while maintaining its original aspect ratio.

## Before
Screenshot_1592995063

## After
Screenshot_1592995342

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: utility/src/main/java/org/oppia/util/parser/UrlImageParser.kt [hunk 1]
```diff
@@ -69,8 +70,18 @@ class UrlImageParser private constructor(
       val drawable = drawableFactory(resource)
       htmlContentTextView.post {
         htmlContentTextView.width {
-          val drawableHeight = drawable.intrinsicHeight
-          val drawableWidth = drawable.intrinsicWidth
+          var drawableHeight = drawable.intrinsicHeight
+          var drawableWidth = drawable.intrinsicWidth
+          val minimumImageSize = context.resources.getDimensionPixelSize(R.dimen.minimum_image_size)
+          if (drawableHeight <= minimumImageSize || drawableWidth <= minimumImageSize) {
```
Comment (veena14cs) [inline utility/src/main/java/org/oppia/util/parser/UrlImageParser.kt:76]: Please add kdoc for this calculation it will be nice.

**Review by veena14cs — APPROVED** [review]
Thanks @rt4914! LGTM.

File: utility/src/main/java/org/oppia/util/parser/UrlImageParser.kt (hunk 1 above)
Comment (rt4914) [reply utility/src/main/java/org/oppia/util/parser/UrlImageParser.kt:76]: Added documentation PTAL

Comment (veena14cs) [reply utility/src/main/java/org/oppia/util/parser/UrlImageParser.kt:76]: Thanks @rt4914.

**Pull request merged**